src/Benchmarks/Benchmarks.csproj

```diff
@@ -16,8 +16,14 @@
     <PackageReference Include="Microsoft.Windows.SDK.Contracts" Version="10.0.18362.2005" Condition="$(UseWinmd) == true"/>
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.3" Condition="$(UseWinmd) == false"/>
   </ItemGroup>
-
+	
   <ItemGroup>
+    <Reference Condition="'$(TargetFramework)' == 'netcoreapp2.0'" Include="$(MSBuildThisFileDirectory)..\Projections\Windows\bin\x64\Release\netstandard2.0\Windows.dll"></Reference>
```

Comment:
This can be simplified if you define a new property like BenchmarkTargetFramework and set it to TargetFramework and then check if it is equal to necoreapp2.0 and if so, reset it to netstandard 2.0.